pkg/api/mimo.go

```diff
@@ -0,0 +1,37 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+type MaintenanceManifestState string
```

Comment:
share we considering using `int` to represent the states? instead of `string`

Reply:
the string is because it'll go into the external API and so we want it to be more readable, see maintenancetype for adminupdates